docs/modules/ROOT/pages/multitoken.adoc

```diff
@@ -0,0 +1,145 @@
+
+- **Uint<128> as value and id type** - Since 256-bit unsigned integers are not supported, the library uses the Compact type `Uint<128>`.
+
+**Features and specifications NOT supported**
+
+- **Events** - Midnight does not currently support events, but this is planned on being supported in the future.
+- **Uint256 type** - There's ongoing research on ways to support uint256 in the future.
+- **Interface** - Compact currently does not have a way to define a contract interface.
+This library offers modules of contracts with free floating circuits; nevertheless, there's no means of enforcing that all circuits are provided.
+- **Batch mint, burn, transfer** - Without support for dynamic arrays, batching transfers is difficult to do without a hacky solution.
+For instance, we could change the `to` and `from` parameters to be vectors.
+This would change the signature and would be both difficult to use and easy to misuse.
+- **Querying batched balances** - This can be somewhat supported.
+The issue, without dynamic arrays, is that the module circuit must use Vector<n> for accounts and ids;
+therefore, the implementing contract must explicitly define the number of balances to query in the circuit i.e.
+
+> ```ts
+> balanceOfBatch_10(
+>    accounts: Vector<10, Either<ZswapCoinPublicKey, ContractAddress>>,
+>    ids: Vector<10, Uint<128>>
+> ): Vector<10, Uint<128>>
+> ```
+> Since this module does not offer mint or transfer batching, balance batching is also not included at this time.
+
+- **Introspection** - Compact currently cannot support contract-to-contract queries for introspection.
+{erc165} (or an equivalent thereof) is NOT included in the contract.
+- **Safe transfers** - The lack of an introspection mechanism means safe transfers of any kind can not be supported.
+
+== Contract-to-contract calls
+
+Contract-to-contract calls are currently not supported in the Compact language.
+Due to this limitation, the current iteration of MultiToken disallows transfers and mints to the `ContractAddress` type.
+Transferring tokens to a contract may result in those tokens being locked forever.
+The MultiToken module, however, does provide `unsafe` circuit variants for users who wish to experiment with sending tokens to contracts.
+
+WARNING: The `unsafe` circuits will eventually be deprecated after Compact supports contract-to-contract calls—meaning
+`transferFrom`, `_mint`, etc. are planned to eventually allow the recipients to be of the `ContractAddress` type.
+
+== Usage
+:extensibility-pattern: xref:extensibility.adoc#the_module_contract_pattern[Module/Contract Pattern]
+:multitoken-mint: xref:/api/multitoken.adoc#MultiTokenModule-_mint[_mint]
+
+Import the MultiToken module into the implementing contract.
+It's recommended to prefix the module with `MultiToken_` to avoid circuit signature clashes.
+
+```typescript
+pragma language_version >= 0.15.0;
+
+import CompactStandardLibrary;
+import openzeppelin-compact/multi-token prefix MultiToken_;
```

Comment:
Using #179 as a reference. Not sure if we want to remove the prefix here too?
```suggestion
import "./node_modules/@openzeppelin-compact/multi-token/MultiToken" prefix MultiToken_;
```

Reply:
> Using https://github.com/OpenZeppelin/compact-contracts/pull/179 as a reference. Not sure if we want to remove the prefix here too?

I propose to remove the prefix just in the API example import declaration (we can discuss in the template proposal)

I say we keep docs consistent instead of adding anticipatory changes. Once we're all in agreement with #179, we can update the rest. It's easy to lose track of things especially when new/different ideas emerge. WDYT?